eve/tests/e2e/test_crypto.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from click.testing import CliRunner
+
+from eve.commands.cmd_crypto import cli
+from eve.config import DEFAULT_COIN, CURRENCY
+
+
+@pytest.fixture
+def runner():
+    return CliRunner()
+
+
+def test_price_returned_with_default_options(runner):
+    result = runner.invoke(cli, ["price"])
```

Comment:
I presume this actually calls the underlying API? - So we're doing a full integration test with the API, compared to a unit test to test just functionality that should not make any network requests.

Reply:
Yes, these are an actual calls to the API.
I was thinking that here we might have 2 types of tests. End to end where we cover big green path scenarios(that is what this PR for) and component/unit where we can mock stuff and cover more)
However, I'm not sure how exactly should I organise neither end to end nor unit/component tests

Reply:
Yeah, I think unit tests can live under `tests/services/` and `e2e` tests like you've here.